githttp.go

```diff
@@ -10,6 +10,14 @@ import (
 	"strings"
 )
 
+type ResolveInfo struct {
+	Path    string
+	Request *http.Request
+}
+
+// Resolve a URL path to a repo location
```

Comment:
This documentation doesn't seem sufficient. As someone who reads this for the first time and tries to use it, I have the following unanswered questions:

- What is URL path? Is it the path component of a URL?
- If so, then why is it provided as a separate `Path` field inside `ResolveInfo`, is it different from `Request.URL.Path`?
- What does repo location mean, is is a system filepath (e.g., "C:\MyRepo" on Windows and "/home/user/MyRepo" on Linux) where the repo is? Or something else?

It'd be helpful to document this better.